backend/src/main/java/com/tourneynizer/tourneynizer/model/MatchChildren.java

```diff
@@ -80,21 +54,6 @@ public Long getTeamChild2() {
         return knownTeamChildren;
     }
 
-    Map<String, String> jsonFirst() {
-        Map<String, String> map = new HashMap<>();
-        map.put("type", teamChild1 != null ? "team" : "match");
-        map.put("id", "" + first());
-        return map;
-    }
-
-    Map<String, String> jsonSecond() {
-        Map<String, String> map = new HashMap<>();
-        map.put("type", teamChild2 != null ? "team" : "match");
-        map.put("id", "" + second());
-        return map;
-    }
-
```

Comment:
Wait so sincec you got rid of this what will be printed for a matchChildren instance? did you want to get rid of the other @JsonIgnores?

Reply:
Oh wait my bad I see the serializer now

Reply:
No, see `MatchChildrenTest.java` and `MatchChildrenSerializer.java`